CHANGES.rst

```diff
@@ -1,32 +1,32 @@
-Changelog of hydxlib



```

Comment:
Ik denk dat de ^M erbij is gekomen door de merge conflicts of niet?
Ik heb ze in ieder geval niet getypt